Updates-2.0/u2-sce-test-wso2is-5.11.0-full-testgrid.yaml

```diff
@@ -0,0 +1,55 @@
+# TestGrid job configuration
+
+version: '0.9'
+emailToList: "builder@wso2.org,shanikaw@wso2.com,niluka@wso2.com,achinij@wso2.com,
+              kanapriya@wso2.com,iam-builder@wso2.com,integration-builder@wso2.com,
+              ashen@wso2.com,jayangak@wso2.com"
+infrastructureConfig:
+  iacProvider: CLOUDFORMATION
+  infrastructureProvider: AWS
+  containerOrchestrationEngine: None
+  includes:
+    - CentOS-7.5
+    - MySQL-5.7
+    - Oracle-SE2-12.1
+    - SQLServer-SE-14.00
+    - Postgres-10.5
+    - Postgres-12.3
```

Comment:
Shall we keep the latest or mostly used by the clients from the two Postgres versions?

Reply:
removed the old version